README.md

```diff
@@ -2,10 +2,12 @@
 Ground Support Equipment Software for the Cornell Rocketry Team including fill station and ground server
 
 ## Ground Server
-To start the ground server, run
-```sudo snap install docker```
-```docker pull zachgarcia42/ground-server-image```
-```docker run -it -d -p 80:80 zachgarcia42/ground-server-image```
+
+First, install Docker for your specific computer. Then, to start the ground server run:
+```shell
+docker pull ghcr.io/cornellrocketryteam/ground-server
+docker run -it -d -p 80:80 ghcr.io/cornellrocketryteam/ground-server
```

Comment:
Should this be give.io/ground-server? The build/push rule defines env.Registry/env.Image_Name which wouldn't include "cornellrocketryteam", right?

Reply:
So I'm pretty sure it actually automatically scopes the image under the user, so in this case cornellrocketryteam. Before, the env.Image_Nae was actually just ground-software (according to [this](https://stackoverflow.com/questions/70700244/in-a-github-actions-workflow-how-can-you-determine-the-organziation-where-the-c) stack overflow post). So it adds the cornellrocketryteam automatically.

Reply:
Cool, thanks for clarifying!